ldb_reader_test.go

```diff
@@ -160,6 +160,78 @@ func TestGetRowsByKeyPrefix(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
```

Comment:
unrelated to the scope of this PR, but reflecting my knowledge acquired by working on: https://segment.atlassian.net/browse/CONFIG-45 and reading the discussion on: https://twilio.slack.com/archives/CB8HZ02J1/p1657738720115539